### PR TITLE
Add CFP delay blog post

### PR DESCRIPTION
**Pull request by ericholscher**
This moves the CFP deadline to Feb 1 and notification back to Feb 15.
We've also gotten our first sponsor,
so I've included them in the mailer as well.

### CODE REVIEW

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+:template: {{year}}/generic.html
+
+.. post:: Jan 22, {{year}}
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+CFP update and grant program reminder
+=====================================
+
+The last few weeks has been pretty rough on us here in the US.
```
Comment (plaindocs) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
The last few weeks have been pretty rough on those of us who are in the US.
```

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Knowing that,
+we have made the decision to move our CFP deadline back a week in order to give you more time to get proposals in.
+We hope to see some new voices in the program this year,
```
Comment (plaindocs) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
We'd love to see some new voices in the program this year,
```

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+given the ability to participate remotely.
```
Comment (plaindocs) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
and we hope that prerecorded talks and fewer timezone restrictions makes that possible.
```

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+CFP deadline moved to February 1st
+----------------------------------
+
+Check out the `Call for Proposals page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/#submit-your-proposal>`_ for more details about what we are looking for.
+Especially if you've not been able to attend due to travel logistics, or if you've not spoken before we'd love to hear your voice.
+The virtual conference format means you can speak from anywhere, though it's ideal if you are close to the local time in Portland.
```
Comment (plaindocs) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
The virtual conference format means you can speak from anywhere, though it's ideal if you can be online and present around your talk slot in Portland time, PST.
```

**Review by plaindocs — COMMENTED** [review]
Thanks for getting this ball rolling @ericholscher

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+.. post:: Jan 22, {{year}}
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+CFP update and grant program reminder
+=====================================
+
+The last few weeks has been pretty rough on us here in the US.
+Knowing that,
+we have made the decision to move our CFP deadline back a week in order to give you more time to get proposals in.
```
Comment (thatdocslady) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
we have made the decision to extend our CFP deadline by a week in order to give you more time to get proposals in.
```

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+We hope to see some new voices in the program this year,
+given the ability to participate remotely.
+
+CFP deadline moved to February 1st
```
Comment (thatdocslady) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
CFP deadline extended to February 1st
```

File: docs/conf/portland/2021/news/cfp-delay.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+----------------------------------
+
+Check out the `Call for Proposals page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/#submit-your-proposal>`_ for more details about what we are looking for.
+Especially if you've not been able to attend due to travel logistics, or if you've not spoken before we'd love to hear your voice.
+The virtual conference format means you can speak from anywhere, though it's ideal if you are close to the local time in Portland.
+
+Submit your talk proposals by February 1st,
```
Comment (thatdocslady) [inline docs/conf/portland/2021/news/cfp-delay.rst]:
```suggestion
Submit your talk proposals by **February 1st**,
```

**Review by thatdocslady — COMMENTED** [review]
Looks great, a few tweaks :)

**Review by mxsasha — APPROVED** [review]

**Pull request merged**